releasenotes/AzureDevOpsPipelineExtension.md

```diff
@@ -0,0 +1,15 @@
+## Azure App Configuration Extension release notes
+
+The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version is printed out to console in the step of running the Azure App Configuration extension: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
```

Comment:
How about "The version of the Azure App Configuration task being used will print to the console when the task is executed."

Reply:
Will update to that